src/components/StatusBadge/StatusBadge.stories.tsx

```diff
@@ -2,28 +2,35 @@ import React from 'react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 
 import StatusBadge from './StatusBadge';
+import { StatusBadgeProps } from '@storybook/addon-interactions/dist/ts3.9/components/StatusBadge/StatusBadge';
+
+const statuses = {
+  unpaid: 'Unpaid',
+  partial: 'Partial',
+  paid: 'Paid',
+};
 
 export default {
   title: 'StatusBadge',
   component: StatusBadge,
   argTypes: {
-    status: { control: 'inline-radio', options: ['UNPAID', 'PARTIAL', 'PAID'] },
+    status: { control: 'inline-radio', options: [statuses.unpaid, statuses.partial, statuses.paid] },
```

Comment:
With the changes suggested in the comment of the `StatusBadge.tsx`, you could tweak this story to make it a bit better.

Here's my suggestion:
```jsx
// 1. Statues now will be lowered-case
const statuses = {
  unpaid: 'unpaid',
  partial: 'partial',
  paid: 'paid',
};

export default {
  title: 'StatusBadge',
  component: StatusBadge,
  argTypes: {
    // 2. Object.values() will save you from hand-writing the options 🤘🏻
    status: { control: 'inline-radio', options: Object.values(statuses) },
  },
} as ComponentMeta<typeof StatusBadge>;
```